core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -92,6 +92,16 @@ private[spark] class Executor(
   private val executorActor = env.actorSystem.actorOf(
     Props(new ExecutorActor(executorId)), "ExecutorActor")
 
+  // Whether to load classes in user jars before those in Spark jars
+  private val userClassPathFirst: Boolean = {
+    val oldKey = "spark.files.userClassPathFirst"
+    val newKey = "spark.executor.userClassPathFirst"
+    if (conf.contains(oldKey)) {
+      logWarning(s"$oldKey is deprecated. Please use $newKey instead.")
```

Comment:
This will log it on the executors, which seems weird since all other configuration warnings are logged at the driver or submission client. Can we just log this warning on the driver side? Or just don't log a deprecation warning always?